### PR TITLE
Update solipsism 

### PR DESCRIPTION
**Pull request by Marenz**
Update solipsism to latest version and use modern way to replace event loop

### CODE REVIEW

**Review by matthias-wende-frequenz — DISMISSED** [review]

**Review by leandro-lucarella-frequenz — DISMISSED** [review]

File: src/frequenz/dispatch/__init__.py [hunk 1]
```diff
@@ -9,7 +9,11 @@
 
 from frequenz.dispatch.actor import DispatchActor, DispatchEvent
 
-__all__ = ["Dispatcher"]
+DispatchReceiver = Receiver[Dispatch]
+"""Type alias for a dispatch receiver."""
+
+DispatchEventReceiver = Receiver[DispatchEvent]
+"""Type alias for a dispatch event receiver."""
```
Comment (llucax) [inline src/frequenz/dispatch/__init__.py]: Is this really related to this commit? If so maybe mention it in the commit message?

**Review by llucax — COMMENTED** [review]
About the starting of the actor, I would add a `initialize()` function that creates a global  `Dispatcher`/actor object (not for this PR).

File: src/frequenz/dispatch/__init__.py (hunk 1 above)
Comment (Marenz) [reply src/frequenz/dispatch/__init__.py]: Isn't this already in the message? Those are the types that i make accessible

Comment (Marenz) [reply src/frequenz/dispatch/__init__.py]: I have rephrased it for your liking

**Comment by llucax** [comment]
Oh, I guess I made a few changes in my PR that were present here. I thought I saw some docs examples being fixed, I think I forgot it was a different PR and was on a sea of catching up so it was hard to keep track and keep everything in my mind, sorry about that. I guess this PR needs some renaming and updating the description. Also, tests are failing :(

**Comment by Marenz** [comment]
It wasn't much work, not a problem ;) 

Updated description

**Review by llucax — APPROVED** [review]
